src/OrbitGl/AccessibleTrack.cpp

```diff
@@ -33,15 +34,29 @@ class FakeTimerPane : public CaptureViewElement {
   }
 
   [[nodiscard]] Vec2 GetPos() const override {
-    CaptureViewElement* track_tab = track_->GetAllChildren()[0];
-    Vec2 pos{track_->GetPos()[0], track_tab->GetPos()[1] + track_tab->GetHeight()};
+    // The element is positioned after the last visible child. We can safely assume there's always
+    // one child due to the track header.
+    CaptureViewElement* last_child = *track_->GetNonHiddenChildren().rbegin();
+    float pos_y = last_child->GetPos()[1] + last_child->GetHeight();
+
+    if (track_->GetNonHiddenChildren().size() == 1) {
+      // If there's only one child, the track only has timers and a header. In this case add the
+      // content margin
+      pos_y += layout_->GetTrackContentTopMargin();
+    } else {
+      // Otherwise, it's a thread track and we need to include the space between panes.
+      // This is really hacky and will go away once this class vanishes, see the TODO on top.
+      pos_y += layout_->GetSpaceBetweenThreadPanes();
+    }
+    Vec2 pos{track_->GetPos()[0], pos_y};
     return pos;
   }
 
   [[nodiscard]] float GetHeight() const override {
     float height = track_->GetHeight();
     float track_header_height = GetPos()[1] - track_->GetPos()[1];
     height -= track_header_height;
+    height -= layout_->GetTrackContentBottomMargin();
```

Comment:
Shouldn't we also count TrackContentTopMargin here?

Reply:
No, it's already accounted for in `GetPos()[1] - track_->GetPos()[1]`

Reply:
Sorry if I wasn't very clear. Currently it's not counted in GetPos(), but it IS counted here.

My suggestion is:
`height -= layout_->GetTrackContentBottomMargin() + layout_->GetTrackContentTopMargin();`

Reply:
But that's incorrect. `GetPos()` already returns the position of the track content, and it contains the header height + top margin. `GetPos()[1] - track_->GetPos()[1]` is everything above the timer pane, including the top margin. So the final height of the timer pane is (track height) - (space above pane) - (bottom margin).

Reply:
oh, you are right. Thanks for explaining it!